compiler/plc2x/src/cli.rs

```diff
@@ -40,6 +40,62 @@ pub fn check(paths: &[PathBuf], suppress_output: bool) -> Result<(), String> {
     Ok(())
 }
 
+pub fn export_json(
```

Comment:
I don't see anything that calls this method. I would have expected this to be called from the `main()` function (in the `bin` folder).